src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php

```diff
@@ -235,7 +235,7 @@ public static function buildExpressions(array $expressions)
             if ($expr instanceof self) {
                 $if = $expr->ifPart;
                 $then = $expr->thenPart;
-                $expressions[$k] = function ($v) use ($if, $then) {
+                $expressions[$k] = function (mixed $v) use ($if, $then) {
```

Comment:
I think we don't need to add types to internal closures: that's going to create merge conflicts but won't help in anyway otherwise

Reply:
Okay, I've removed the `mixed` type from those closures.